assets/javascripts/discourse/components/campaign-banner.js.es6

```diff
@@ -0,0 +1,143 @@
+import { action } from "@ember/object";
+import Component from "@ember/component";
+import discourseComputed, { observes } from "discourse-common/utils/decorators";
+import { inject as service } from "@ember/service";
+import User from "discourse/models/user";
+import { Promise } from "rsvp";
+
+export default Component.extend({
+  router: service(),
+  dismissed: false,
+  loading: false,
+  showContributors: false,
+  contributors: [],
```

Comment:
You need to initialize ember arrays in `init()` methods, otherwise all components point at the same array instance.